cmd/plugin/cli/install.go

```diff
@@ -250,5 +259,8 @@ func setInstallValues(cmd *cobra.Command, config *apiv1.KubectlStorageOSConfig)
 	config.Spec.Install.PortalTenantID = viper.GetString(installer.PortalTenantIDConfig)
 	config.Spec.Install.PortalAPIURL = viper.GetString(installer.PortalAPIURLConfig)
 	config.InstallerMeta.StorageOSSecretYaml = ""
+	config.Spec.IncludeLocalPathProvisioner = viper.GetBool(installer.IncludeLocalPathProvisionerFlag)
+	config.Spec.Install.LocalPathProvisionerYaml = viper.GetString(installer.LocalPathProvisionerYamlFlag)
+
```

Comment:
This comment seemed to get lost in my last review, so adding again:

Should be `installer.IncludeLocalPathProvisionerConfig` and `installer.LocalPathProvisionerYamlFlag`